### PR TITLE
[ACM-15258] Update VM page error message

### PR DESCRIPTION
**Pull request by zlayne**
Updates the error message on the Virtual Machines page when search is unavailable.

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2957,6 +2956,7 @@
   "To run automation manually, select \"Disabled\" and check the \"Manual run\" checkbox.)": "To run automation manually, select \"Disabled\" and check the \"Manual run\" checkbox.)",
   "To search for a keyword, type the word in the search box.": "To search for a keyword, type the word in the search box.",
   "To search for resources with a given property value, type or select the property name from the autocomplete list. Then type or select the value for the selected property filter.": "To search for resources with a given property value, type or select the property name from the autocomplete list. Then type or select the value for the selected property filter.",
+  "To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.",
```
Comment (dockerymick) [inline frontend/public/locales/en/translation.json:2959]: Based on our convo in Slack about the user installing RHACM from the Catalog. Are users able to enable RHACM from MCE? If no, lets go with "install". 
```suggestion
  "To view managed virtual machines, you must install Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must install Red Hat Advanced Cluster Management.",
```

OR 

```suggestion
  "To view managed virtual machines, you must enable Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must enable Red Hat Advanced Cluster Management.",
```

File: frontend/src/routes/Infrastructure/VirtualMachines/VirtualMachinesPage.tsx [hunk 2]
```diff
@@ -140,11 +140,13 @@ function VirtualMachineTable() {
...
         <EmptyStateIcon icon={ExclamationCircleIcon} color={'var(--pf-global--danger-color--100)'} />
         <Title size="lg" headingLevel="h4">
-          {t('Unable to display VirtualMachines')}
+          {t('Unable to display virtual machines')}
         </Title>
         <EmptyStateBody>
           <Stack>
-            <StackItem>{t('Enable search to view all managed VirtualMachines.')}</StackItem>
+            <StackItem>
+              {t('To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.')}
```
Comment (dockerymick) [inline frontend/src/routes/Infrastructure/VirtualMachines/VirtualMachinesPage.tsx:148]:
```suggestion
              {t('To view managed virtual machines, you must install Red Hat Advanced Cluster Management.')}
```

OR

```suggestion
              {t('To view managed virtual machines, you must enable Red Hat Advanced Cluster Management.')}
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (dockerymick) [reply frontend/public/locales/en/translation.json:2959]: Ok, so would it be more accurate to say, "....you must enable Search for Red Hat Advanced Cluster Management."

Comment (zlayne) [reply frontend/public/locales/en/translation.json:2959]: Yea, I think that best captures the error state.

File: frontend/src/routes/Infrastructure/VirtualMachines/VirtualMachinesPage.test.tsx [hunk 3]
```diff
@@ -233,8 +233,12 @@ describe('VirtualMachinesPage Page', () => {
     // This wait pauses till apollo query is returning data
     await wait()
     // Test that the component has rendered errors correctly
-    await waitFor(() => expect(screen.queryByText('Unable to display VirtualMachines')).toBeTruthy())
-    await waitFor(() => expect(screen.queryByText('Enable search to view all managed VirtualMachines.')).toBeTruthy())
+    await waitFor(() => expect(screen.queryByText('Unable to display virtual machines')).toBeTruthy())
+    await waitFor(() =>
+      expect(
+        screen.queryByText('To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.')
```
Comment (dockerymick) [inline frontend/src/routes/Infrastructure/VirtualMachines/VirtualMachinesPage.test.tsx:239]:
```suggestion
        screen.queryByText('To view managed virtual machines, you must enable Search for Red Hat Advanced Cluster Management.')
```

**Pull request merged**